components/brave_shields/browser/ad_block_base_service.cc

```diff
@@ -110,10 +110,6 @@ AdBlockBaseService::AdBlockBaseService(BraveComponent::Delegate* delegate)
       weak_factory_(this) {}
 
 AdBlockBaseService::~AdBlockBaseService() {
-  Cleanup();
```

Comment:
Calling virtual functions in destructors is generally a bad practice, and also `Cleanup` is not actually used